src/Settings_Helper.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace Tribe\Extensions\Example;
+
+use Tribe__Main;
+
+if ( ! class_exists( 'Settings_Helper' ) ) {
```

Comment:
Needs to be namespace aware.

Reply:
Good catch. Added to main extension file's check as well.